src/bug-filing/types/bug-filing-service.ts

```diff
@@ -6,6 +6,12 @@ import { ReactSFCWithDisplayName } from './../../common/react/named-sfc';
 import { CreateIssueDetailsTextData } from './../../common/types/create-issue-details-text-data';
 import { SettingsFormProps } from './settings-form-props';
 
+export type IssueFilingUrlProvider<TSettings> = (
```

Comment:
Do we have a team standard about whether we prefer the `T` prefix for type parameters? It's surprising to me that we'd want to avoid `I` but keep `T`. I note that `BugFilingService<Settings>` on line 15 doesn't use `T`.

I couldn't care less whether we use `T` or not, personally, but it'd be nice to be consistent.

Reply:
That's a good point. I don't think we have talk about it. I'll got with consistency at the PR level and I guess we should check the current state is right now for this names.